airwallex/src/main/java/com/airwallex/android/AirwallexStarter.kt

```diff
@@ -88,6 +89,7 @@ class AirwallexStarter {
             paymentResultListener: Airwallex.PaymentResultListener,
         ) {
             setupAnalyticsLogger(session)
+            AirwallexRisk.log(AirwallexRisk.Events.TRANSACTION_INITIATED, "")
```

Comment:
Shall we just do `AirwallexRisk.log(AirwallexRisk.Events.TRANSACTION_INITIATED)` to avoid extra empty string to pollute backend log or align with iOS ("payment_sheet")?